lib/src/pdf_data_range_transport.dart

```diff
@@ -17,14 +17,13 @@ part of pdfjs;
 abstract class PDFDataRangeTransport {
   JsObject _jsInternal;
 
-  PDFDataRangeTransport(int length, Uint8List initialData) {
-    _jsInternal = JsObject(PDFJS.versionSafePdfJsContext['PDFDataRangeTransport'] as JsFunction, [
+  PDFDataRangeTransport(int length, Uint8List initialData) :
```

Comment:
See note above in doc_init_parameters. Area of note involves fetching the data for the pdf via the `CustomDataRangeTransport`